draft-ietf-quic-transport.md

```diff
@@ -4150,8 +4154,10 @@ the client included in the Source Connection ID of the Initial packet.
 
 The server includes a connection ID of its choice in the Source Connection ID
 field.  This value MUST not be equal to the Destination Connection ID field of
-the packet sent by the client.  The client MUST use this connection ID in the
-Destination Connection ID of subsequent packets that it sends.
+the packet sent by the client.  A client MUST discard a Retry packet that
+contains a Source Connection ID field that is identical to the Destination
+Connection ID field of its Initial packet.  The client MUST use this connection
+ID in the Destination Connection ID field of subsequent packets that it sends.
 
```

Comment:
Post merge, but "The client MIST use this connection ID" reads as if it refers to the invalid duplicate ID.

Reply:
https://github.com/quicwg/base-drafts/pull/3153